ui/page.go

```diff
@@ -299,8 +297,11 @@ func (win *Window) addPages(decredIcons map[string]image.Image) {
 		changePage:              win.changePage,
 		setReturnPage:           win.setReturnPage,
 		refreshWindow:           win.refresh,
+		toastList:               layout.List{Axis: layout.Vertical},
```

Comment:
I don't think godcr has a use case where there will be more than one notification at once. That means there's no need to create a list layout.

Reply:
If only display toast at once app only need single struct of toast, I don't think app need slice of toasts